extensions/mssql/src/sqlProjects/sqlProjectsService.ts

```diff
@@ -415,7 +415,7 @@ export class SqlProjectsService extends BaseService implements mssql.ISqlProject
 	/**
 	 * Add a SQL object script to a project
 	 * @param projectUri Absolute path of the project, including .sqlproj
-	 * @param path Path of the script, including .sql, relative to the .sqlproj
+	 * @param path Path of the script, including .sql and .publish.xml, relative to the .sqlproj
```

Comment:
Hmm, maybe these comments should be "including extension" since we're not limited to just those two file types.  And also perhaps not call it "script" here.

> Path of the file, including extension, relative to the .sqlproj

Reply:
Updated.